pkgs/development/tools/rust/cargo-rr/default.nix

```diff
@@ -9,16 +9,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-rr";
-  version = "0.1.3";
+  version = "v0.2.0";
```

Comment:
```suggestion
  version = "0.2.0";
```

Reply:
The reason for this is because otherwise the metadata of the package will not match what is expected. So never add `v` before the version number, always add it only where needed.

Reply:
I fixed it.